product_supplierinfo_for_customer/tests/test_product_supplierinfo_for_customer.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2015 OdooMRP team
+# Copyright 2015 AvanzOSC
+# Copyright 2015 Tecnativa
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+import odoo.tests.common as common
+
+
+class TestProductSupplierinfoForCustomer(common.TransactionCase):
```

Comment:
Consider changing this to SavepointCase. As you have a lot of test cases it will be a lot quicker

Reply:
I suppose this is an optional change, right?

Reply:
Well, it's a well received optimization. The problem both have is that when testing on parallel with other modules that tests reports, both will hang, so `HttpCase` is the only that saves that problem.